app/cisc/cisc.go

```diff
@@ -0,0 +1,312 @@
+package main
+
+import (
+	"os"
+
+	"encoding/hex"
+
+	"path"
+
+	"io/ioutil"
+	"strings"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/services/identity"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+/*
+Cisc is the Cisc Identity SkipChain to store information in a skipchain and
+being able to retrieve it.
+
+This is only one part of the system - the other part being the cothority that
+holds the skipchain and answers to requests from the cisc-binary.
+*/
+
+var configFile string
+var sshDir string
+var sshConfig string
```

Comment:
Does all the 3 above global variables have to be globals ... ? After you then have methods like `loadConfigfile()` which takes no arguments. In my head, I would have expect the method to take a string to the config file as an argument ... 
We've had so much trouble with globals so if we can avoid them that'd be best. I'm not proficient enough with the code to say if it's possible or not, I let your judgement decide :D